conbench/api/_google.py

```diff
@@ -1,44 +1,72 @@
 import json
+import logging
 import os
+import time
 
 import flask as f
 import requests
 
+from ..config import Config
 
-def get_google_config():
+log = logging.getLogger(__name__)
+
+# Rely on INTENDED_BASE_URL to have a trailing slash.
+OIDC_CALLBACK_URL = Config.INTENDED_BASE_URL + "api/google/callback"
+
+
+def get_oidc_config():
     client_id = os.environ.get("GOOGLE_CLIENT_ID", None)
     client_secret = os.environ.get("GOOGLE_CLIENT_SECRET", None)
-    discovery_url = "https://accounts.google.com/.well-known/openid-configuration"
+    discovery_url = Config.OIDC_ISSUER_URL + "/.well-known/openid-configuration"
```

Comment:
I think this change shows pretty well what this PR is mostly about.

Reply:
This _is_ a fun minimal change to go from single source -> OIDC I wish all abstraction PRs were as simple as this! 🎉